.dockerignore

```diff
@@ -0,0 +1,7 @@
+.vagrant
+.git/objects/*
```

Comment:
This was done to enable the git sub-modules fetching of 3rd party stuff, including js, but to reduce the size of the context send to docker. I'll have to check all of this in light of your 404ing js below, because those items should have been fetched, so it sounds like there is a problem somewhere else.

Reply:
@cdrini I rechecked this by setting it to `.git` and the `make` command fails on https://github.com/internetarchive/openlibrary/blob/495c389c31f7d6b6112100113ac70107ad1139d8/Makefile#L32  which runs as part of `git all`, so for now `.git/objects/*` excludes as much of the git cruft from the Docker build context as I could manage. There may be a way to remove  this, but it will involve digging into the `git` task in the Makefile